src/js/tracks/text-track-settings.js

```diff
@@ -321,17 +323,12 @@ class TextTrackSettings extends Component {
    * @method restoreSettings
    */
   restoreSettings() {
-    let err;
     let values;
 
     try {
-      [err, values] = safeParseTuple(window.localStorage.getItem('vjs-text-track-settings'));
```

Comment:
Also, the other benefit of safejsonparse is encapsulation. Yeah, not much help here but I think it does help in general. Supposedly, try/catch don't hurt much anymore in the latest and greatest js engines but older engines still have issues with deoping the entire function.